vips/convolution.c

```diff
@@ -4,6 +4,6 @@ int gaussian_blur_image(VipsImage *in, VipsImage **out, double sigma) {
 	return vips_gaussblur(in, out, sigma, NULL);
 }
 
-int sharpen_image(VipsImage *in, VipsImage **out, double sigma, double x1, double m2, VipsSharpenMode mode) {
-	return vips_sharpen(in, out, "sigma", sigma, "x1", x1, "m2", m2, "mode", mode, NULL);
+int sharpen_image(VipsImage *in, VipsImage **out, double sigma, double x1, double m2) {
```

Comment:
If we merge this without the sharpen mode parameter, it will break IM

Reply:
I'll reinstate it in master after merge,

Reply:
Let's reinstate it here and then merge, that's the right way to work